examples/transient_examples/PyFstat_example_short_transient_grid_search.py

```diff
@@ -43,16 +45,15 @@
         Alphas=Alphas,
         Deltas=Deltas,
         tref=data.tref,
-        BSGL=False,
+        BSGL=BSGL,
     )
     search1.run()
     search1.print_max_twoF()
-
     search1.plot_1D(xkey="F0", xlabel="freq [Hz]", ylabel="$2\\mathcal{F}$")
 
     print("with t0,tau bands:")
     search2 = pyfstat.TransientGridSearch(
-        label="tCW",
+        label="tCW" + ("-BSGL" if BSGL else ""),
```

Comment:
```suggestion
        label="tCW" + ("_BSGL" if BSGL else ""),
```
Any particular reason for your choice?

Reply:
no